data/phoible-references.bib

```diff
@@ -42457,3 +42457,23 @@ @book{heath2015togokan
 	Title = {A Grammar of Togo Kan (Dogon language family, Mali)},
 	Url = {https://deepblue.lib.umich.edu/handle/2027.42/123066},
 	Year = {2015}}
+
+@article{rose2021mojenotrinitario.pdf,
```

Comment:
@Alessioryan -- there shouldn't be the ".pdf" in the ID (like the rest of the entries). Please remove.

Reply:
Done! I misunderstood a comment in a previous email but both are fixed now.